### ISSUE TITLE
Attribution Comments Missing from SVGs?

### ISSUE THREAD

**Issue by GuyPaddock** [FortAwesome/Font-Awesome#15580]
**Describe the bug**
According to the documentation:
> Downloaded Font Awesome Free files already contain embedded comments with sufficient attribution, so you shouldn't need to do anything additional when using these files normally.

But, looking at SVGs like [user.svg](https://raw.githubusercontent.com/FortAwesome/Font-Awesome/master/svgs/solid/user.svg) or [building.svg](https://raw.githubusercontent.com/FortAwesome/Font-Awesome/master/svgs/solid/building.svg) there's no comment at all in the SVG with the attribution.

**To Reproduce**
Open the raw SVG content for an icon.

**Expected behavior**
The SVG should contain a comment with attribution, as indicated in the docs.

**Version and implementation**
Version: 5.x
- [x] SVG Sprites

**Bug report checklist**
- [x] I have filled out as much of the above information as I can
- [ ] I have included a test case because my odds go _way_ up that the team can fix this when I do
- [x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by apennell** [comment]
Plus one on this! I'm trying to figure out how to properly give attribute to the svg but so far haven't been able to find how to do that. I'm not plopping the file straight in my code, I'm using the path and viewBox for a dynamic Icon component. Is it ok if I put something like this at the top of the files?:
```js
/**
 * Adapted from Font Awesome arrow-right: https://fontawesome.com/icons/arrow-right?style=solid,
 * which is licensed under the Creative Commons Attribution 4.0 International license:
 * https://creativecommons.org/licenses/by/4.0/
 */
```

**Comment by tagliala** [comment]
@robmadole I think we already have discussed the attribution when using individual `.svg` files, but at the moment I'm not able to find the original question

Any chance to clarify one more time?

**Comment by cozimacode** [comment]
Same here. Any comment from official FA team would be great. Could we add the comment our self as mentioned by @apennell ?

**Comment by tagliala** [comment]
This has been fixed in version 5.15.0

https://github.com/FortAwesome/Font-Awesome/blob/f0b80521fddaa2a2e40b00576a5b64dfb8fe7301/svgs/regular/address-book.svg#L2-L5

Closing here
